### ISSUE TITLE
Getting started 

### ISSUE THREAD

**Issue by sign-out** [wireapp/wire-desktop#613]
I'm interested in working on the wire client(Linux specifically but I imagine most stuff is universal).
where would you recommend one start familiarizing them selves with the code base? and how about is there any recommended tutorials or documentation for setting up a development environment for wire specifically ?
Is there documentation on the organization of the code? and where could I fine other documentation on the code?

 I imagine some of this I could answer with a google search but I would like your option and suggestions

TL;DR noob asking dumb questions but want help with: where docs are on seeing up dev environment and the project in general , perhaps also the best place a noob could help

**Comment by lipis** [comment]
It depends how deep you want to go. The desktop client is a wrapper built using [Electron](https://electron.atom.io), so the best place to start the [Electron Docs](https://electron.atom.io/docs/). But get ready for lots of JavaScript and if you haven't play with JavaScript before, then let's wait for @bennyn (or anyone else) to provide us with more links and tutorials on where is the best place to start these days.

**Comment by bennyn** [comment]
@sign-out: As long as we don't have any written documentation I recommend to follow the progress of our [API client](https://github.com/wireapp/wire-web-api-client). It's easy to understand and provides a [demo page](https://wireapp.github.io/wire-web-api-client/) to play around with our services. 😃 

There is also a [core messaging library](https://github.com/wireapp/wire-web-core-messaging) which will be helpful for third-party developers but it's work in progress. 🚧 👷
